src/chainparams.cpp

```diff
@@ -133,7 +133,9 @@ class CMainParams : public CChainParams
         // Nov, 13 2017 hard fork
         consensus.daaHeight = 504031;
         // May, 15 2018 hard fork
-        consensus.may2018activationTime = 1526400000;
+        consensus.may2018Height = 530359;
+        // May, 15 2018 hard fork
+        consensus.nov2018ActivationTime = 1542300000;
```

Comment:
This can be similarly factored out, please have a look at #1311.

Reply:
yes we could do it, wonder if it is worth doing cause we are going to change it again just after Nov activation to move it to an height based param. Once we do this we basically have different values for each net...

Reply:
I see. I was looking at the may stuff, which sat there just like that, though. No strong opinion, do as you wish.

Reply:
me neither. I'm ok either way. Let's see what others think about it.